model/event.go

```diff
@@ -39,14 +38,9 @@ func EventsFromFlow(flowEvents []flow.Event) ([]Event, error) {
 }
 
 func parseEvent(event flow.Event) (Event, error) {
-	values := make([]string, len(event.Value.Fields))
-	for j, field := range event.Value.Fields {
-		encoded, _ := jsoncdc.Encode(field)
-		values[j] = string(encoded)
-	}
-
+	// do events have more than one value, this seemed to have changed from previous version
```

Comment:
No they shouldn't.  events hold only one cadence event value, but that can have multiple fields.

Reply:
thanks, stumbling my way through upgrading libraries.